tssc/__main__.py

```diff
@@ -69,6 +71,87 @@ def parse_yaml_or_json_file(yaml_or_json_file):
 
     return parsed_file
 
+def parse_config_files(files):
+    """Parses and merges a list of TSSC configuration files.
+
+    Given a list of files and directories parses all of the child files
+    as TSSC configuration files merges them together.
+
+    Notes
+    -----
+    If any files have duplicate (deep) keys then will throw an error.
+
+    Parameters
+    ----------
+    files : array
+       Array of files and/or directories that should be parsed as TSSC configuration files
+       and merged into one configuration object.
+
+    Returns
+    -------
+    dict
+        TSSC configuration created by merging all of the given files.
+
+    Raises
+    ------
+    ValueError
+        If any of the given file can not be parsed as YAML or JSON.
+        If any of the given files have duplicate (deep) keys.
+    """
+    tssc_config = {}
+    merged_files = []
+    for file in files:
+        # if file is a dir add all of the recursive files under that dir to the list of files
+        # to merge and parse
+        if os.path.isdir(file):
+            dir_files = glob.glob(file + '/**', recursive=True)
+            for dir_file in dir_files:
+                if os.path.isfile(dir_file):
+                    files.append(dir_file)
+        else:
+            parsed_file = parse_yaml_or_json_file(file)
+
+            if not 'tssc-config' in parsed_file:
+                print_error("specified -c/--config-file must have a 'tssc-config' attribute")
```

Comment:
This message should be updated to reference `--config` instead of `--config-file`